DOC-1589: Update branding attribution admonition (#2283)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/branding.adoc

Use the `+branding+` option to disable the "*Powered by Tiny*" link displayed in the status bar for link:{legalpages}/attribution-requirements/[product attribution].

IMPORTANT: Product attribution is required for free users utilizing the Tiny Cloud infrastructure.

> For those using TinyMCE under the MIT license your support for an open source project would be appreciated by keeping attribution turned on in your editor.

Reviewer: ```suggestion
If using TinyMCE under the MIT license, then support for the open source project would be appreciated by keeping attribution turned on in your editor.
```
Pronouns and such shouldn't be used in technical documentation. I'm also not exactly sure what you were trying to say here but hopefully this is the same thing.
